### DISCUSSION TITLE
how to make stable diffusion webui image?

### DISCUSSION THREAD

**Discussion by andylou-tech** [AbdBarho/stable-diffusion-webui-docker#644] [Q&A]
how to just make stable diffusion webui image? 
 the dockfile is locate services/AUTOMATIC1111/Dockerfile?

**Comment by DevilaN** [comment]
It is described in the first section of README: Setup.
I'll leave link here in case you miss it: https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Setup
